src/components/ToastNotification/ToastNotification.tsx

```diff
@@ -62,22 +64,37 @@ export const ToastNotification = ({
     className,
   );
 
+  // if both are defined, temporarily prefer the original (misspelled) prop for now, but throw a warning
+  const tempDismissType =
+    dissmissType && dismissType ? dissmissType : dismissType;
```

Comment:
Won't this always return undefined when `dismissType` is undefined, and thus break existing code that only passes the old `dissmissType` prop?

Reply:
This part was a bit tricky to parse, but tl;dr it won't break in this way:

* `dismissType` has a default value of `'manual'` in the function signature
* b/c of that, it's guaranteed to have some value (and a type check to prevent invalid values)
* previously `dissmissType` had that default valued but no longer has one

| `dismissType` | `dissmissType` | Result |
|--------|--------|--------|
| manual | undefined | manual |
| auto | undefined | auto |
| manual | manual | manual |
| manual | auto | auto | 

The intent is that if someone intentionally used `"auto"` with the old prop, that value would be used (and they'd get that console error). Without a check like this, the old prop's value would be replaced with the new prop's default value

Reply:
I see. So the `&& dismissType` doesn't actually do anything since it's always true? It may be worth pointing that out in the comment, but not a big deal if support for the old prop will be removed soon anyway.

Reply:
Great point; I add this logic before changing the default value, so the check **is** redundant. Since we are here, i'll remove that truthy value from the expression. thanks for the review!